argilla/docs/getting_started/how-to-configure-argilla-on-huggingface.md

```diff
@@ -0,0 +1,128 @@
+---
+description: Configure Argilla on Hugging Face Spaces
+title: Hugging Face Spaces Settings
+---
+
+This section details how to configure and deploy Argilla on Hugging Face Spaces. It covers:
+
+- Persistent storage
+- How to deploy Argilla under a Hugging Face Organization
+- How to configure and disable HF OAuth access
+- How to use Private Spaces
+
+!!! tip "Looking to get started easily?"
+    If you just discovered Argilla and want to get started quickly, go to the [Quickstart guide](quickstart.md).
+
+## Persistent storage
+In the Space creation UI, persistent storage is set to `Small PAID`, which is a paid service, charged per hour of usage.
+
+**Spaces get restarted due to maintainance, inactivity, and every time you change your Spaces settings**. Persistent storage enables Argilla to save to disk your datasets and configurations across restarts.
+
+!!! warning "Ephimeral FREE persistent storage"
+    Not setting persistent storage to `Small` means that **you will loose your data when the Space restarts**.
```

Comment:
Maybe add a reference about the persistent storage warning in the UI and how to disable it? https://docs.argilla.io/en/latest/getting_started/installation/configurations/server_configuration.html#hugging-face